index.rb

```diff
@@ -26,6 +26,27 @@ def show(**kwargs)
 today = Date.today
 venues = []
 
+venues << Venue.new(:name => 'Cornerstone (Berkeley)', :link => 'https://cornerstoneberkeley.com/music-venue/') do
+  URI.open(link) do |html|
+    Nokogiri(html).css('article.list-view-item').map do |article|
```

Comment:
I think you could use `filter_map` for this now: https://ruby-doc.org/core-3.0.2/Enumerable.html#method-i-filter_map